### ISSUE TITLE
message "Auto creating new users is disabled"

### ISSUE THREAD

**Issue by sir-andreas** [pulsejet/nextcloud-oidc-login#225]
I setup Nextcloud (26.0, I know that it is not officially supported yet) with Authelia and everything seems to work so far. Only problem is that if I login with a new user in Authelia that does not exist in NC yet I get the following error:

> "Auto creating new users is disabled"

I already set 
`'oidc_login_disable_registration' => false`

but I still does not work. 

BR

**Comment by pulsejet** [comment]
Did you try a reboot?

**Comment by sir-andreas** [comment]
> Did you try a reboot?

I did a complete new creation of the docker containers of nextcloud and authelia. But I did not reboot the host which would require a downtime for other services as well. 
From my perspective that should be enough, isn't it?

**Comment by pulsejet** [comment]
Yes it should be. No idea what's happening here, maybe a typo somewhere? There are basically just these three lines where the error is thrown. You could try poking here.
https://github.com/pulsejet/nextcloud-oidc-login/blob/049a4cbfe9157deee84cb8c4b678b2b96c9f1c32/lib/Service/LoginService.php#L285-L287

**Comment by sir-andreas** [comment]
Hi, thanks again for the quick anser. I got it now. 

Very embarrassing reason by the way: I copy/pasted some code for the configuration and added it to the config.php
While I was always playing around with the 'oidc_login_disable_registration' => false setting I did not recognize that another line was active hidden between some comments setting
'oidc_login_disable_registration' back to true again. 

I'm sorry for the hassle.
